src/coreclr/jit/compiler.h

```diff
@@ -9184,14 +9151,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         {
             return true;
         }
-        if (JitConfig.JitStressEvexEncoding() && compOpportunisticallyDependsOn(InstructionSet_AVX512F_VL))
```

Comment:
Why is this removed?

Reply:
Shouldn't have been, was a refactoring mistake

Reply:
Fixed.